README.md

```diff
@@ -27,7 +27,7 @@ Initial backends will include ADIOS and HDF5.
 
 // ...
 
-auto s = openPMD::Series("output_files/");
+auto s = openPMD::Series::read("output_files/data00000%T.h5");
```

Comment:
that's weird somehow... 0-padding shall be allowed in `%T`

Reply:
Let me get this right: You would be prefer ```data%T.h5```?
That is still valid and obtains the same files.

I guess I tried to be too smart in the introductory example by showing you can load a limited subset of your series.